lib/rubocop/cop/layout/empty_line_between_defs.rb

```diff
@@ -76,28 +76,40 @@ module Layout
       #   # good
       #   class ErrorA < BaseError; end
       #   class ErrorB < BaseError; end
-      #   class ErrorC < BaseError; end
```

Comment:
Made it more concise by only listing two after each other. The third is not needed for these examples.